javascript/javascript1/week1/age.js

```diff
@@ -0,0 +1,69 @@
+let yearOfBirth = 1995;
+let yearFuture = 2027;
+let age = yearOfBirth - yearFuture;
+console.log("You will be " + age + "years old in " + yearFuture + ".");
+
+let dogYearOfBirth = 2010;
+let dogYearFuture = 2021;
+let dogYear = dogYearFuture - dogYearOfBirth;
```

Comment:
I think the trick is that some people believe that one human year is equal to 7 dog years, so you need to do some math here to get 'right' answer.